src/alt.js

```diff
@@ -494,6 +501,7 @@ class Alt {
 
   addActions(name, ActionsClass) {
     this.actions[name] = this.createActions(ActionsClass)
+    return this.actions[name]
```

Comment:
That's not how it's meant to be used though. addActions adds the action to your alt instance and returns nothing. You probably just want to create an action. That returns.